include/deal.II/distributed/tria_base.h

```diff
@@ -201,6 +202,18 @@ namespace parallel
     const std::set<types::subdomain_id> &
     level_ghost_owners() const;
 
+    /**
+     * Return partitioner for the global cell indices.
+     */
+    const Utilities::MPI::Partitioner &
+    global_cell_index_partitioner() const;
```

Comment:
I don't think I understand what the function is about. Can you be more explicit in the documentation? Same for the next one.